Dockerfile

```diff
@@ -0,0 +1,67 @@
+# This is a multi-stage build which requires Docker 17.05 or higher
+FROM python:3.7-alpine as molecule-builder
+
+WORKDIR /usr/src/molecule
+
+ENV PACKAGES="\
+    gcc \
+    git \
+    libffi-dev \
+    make \
+    musl-dev \
+    openssl-dev \
+    "
+RUN \
+    apk update && \
+    apk add --update --no-cache ${PACKAGES}
+
+ADD . .
+RUN \
+    pip wheel \
+    -w dist \
+    .[azure,docker,docs,ec2,gce,lxc,openstack,vagrant,windows]
+
+# ✄---------------------------------------------------------------------
+# This is an actual target container:
+
+FROM python:3.7-alpine
```

Comment:
Any reason not to use the latest 3.9 (AFAIK)?

Reply:
None, but why not use it if it's already there and stable.